site/docs/releases.md

```diff
@@ -56,7 +56,38 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
   ...
 </dependencies>
 ```
-## 0.12.0 Release Notes
+
+## 0.12.1 Release Notes
+
+Apache Iceberg 0.12.1 was released on November 8th, 2021.
+
+Note that the spark3-runtime jar supports Spark 3.0 and 3.1. Spark 3.2 support will come in the next release or can be tried out with the nightly SNAPSHOT release built from master.
```

Comment:
This has come up so many times that I figured it was worth mentioning. We'll still get asked, so I'm open to removing it.

Reply:
If you mention a snapshot release, we need to make sure that we state it is for developer testing only. We can't recommend anyone actually use unreleased code (ASF guidelines).

Reply:
Spoke offline and agreed to remove entirely.